src/hub.js

```diff
@@ -308,8 +308,24 @@ const onReady = async () => {
     }
   };
 
+  const getPlatformUnsupportedReason = () => {
+    if (typeof RTCDataChannelEvent === "undefined") {
```

Comment:
Probably more correct to check `RTCPeerConnection.prototype.createDataChannel`

Reply:
sounds correct to me, based on this thread: https://wpdev.uservoice.com/forums/257854-microsoft-edge-developer/suggestions/17929639-rtc-data-channels

Reply:
MDN says this API is supported in Edge https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/createDataChannel#Browser_compatibility

Reply:
MDN is lying. I submitted a PR to fix that. https://github.com/mdn/browser-compat-data/pull/1940
You're right, the webrtc-adapter we use is polyfilling createDataChannel on Edge when it shouldn't. I guess we can check for the event class, since it works.